projects/storefrontstyles/scss/components/cds/merchandising/carousel/_merchandising-product-carousel.scss

```diff
@@ -0,0 +1,5 @@
+@import '../../../product/carousel/product-carousel';
```

Comment:
this file should be with the other carousels in the storefrontstyles lib

Reply:
moved into the other carousel folder

Reply:
I guess this was miscommunication on our side. Apologies for this, we should've consulted.